action.yaml

```diff
@@ -39,7 +39,7 @@ runs:
         fi
 
     - name: Get Compose
-      uses: ./get-compose-action
+      uses: getsentry/self-hosted/get-compose-action@master
```

Comment:
Actually, this should have been `${{ github.action_path }}get-compose-action` not sure how I missed this (or maybe a merge error). I'll submit a PR that does this and see if it works or fails.

Reply:
Ah remembered now, can't do what I want due to: https://github.com/orgs/community/discussions/27048

Reply:
I tried that one in this PR but discovered it didn't work haha